sdk/tests/conformance2/vertex_arrays/vertex-array-object.html

```diff
@@ -476,17 +476,22 @@
     // delete the color buffers AND the position buffer.
     gl.bindVertexArray(null);
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      gl.deleteBuffer(colorBuffers[ii]);
-      gl.deleteBuffer(elementBuffers[ii]);
```

Comment:
If we want to delete element buffers too, shouldn't we check if they're still valid? And then make sure they're invalid after the draw call?

Also, this loop only runs three times, but we have 4 separate VAOs with their own element buffers, should we delete for the fourth one as well?